core/src/components/range/range.tsx

```diff
@@ -449,7 +449,7 @@ export class Range implements ComponentInterface {
    */
   private onEnd(detail: GestureDetail | MouseEvent) {
     const { contentEl, initialContentScrollY } = this;
-    const currentX = (detail as GestureDetail).currentX || (detail as MouseEvent).clientX;
+    const currentX = (detail as GestureDetail).currentX ?? (detail as MouseEvent).clientX;
```

Comment:
The problem here was that `detail.currentX` can evaluate to `0` and `0 || detail.clientX` will treat the `0` as falsy and return the `detail.clientX` which is `undefined`. 

Instead we wanted to take the `currentX` or take the `clientX` if the value is null/undefined.